pkg/services/cloudmigration/api/api.go

```diff
@@ -364,19 +364,28 @@ func (cma *CloudMigrationAPI) GetSnapshot(c *contextmodel.ReqContext) response.R
 		return response.ErrOrFallback(http.StatusBadRequest, "invalid snapshot uid", err)
 	}
 
+	// parse and validate query params
+	page, lim, col, dir := c.QueryInt("resultPage"), c.QueryInt("resultLimit"), c.Query("resultSortColumn"), c.Query("resultSortOrder")
+	if lim == 0 {
```

Comment:
Could we make sure the limit is between acceptable bounds? 0 <= lim <= 10000? And what happens with a negative limit?